package com.Api1.API1.Exception;

    public class RuntimeExceptionCPF extends  InternalError{

        public RuntimeExceptionCPF(String message) {
            super(message);
        }
//sera que funciona??

}
